src/i18n/de.json

```diff
@@ -129,7 +129,70 @@
         "verein-home-card-donate-button": "Ich möchte spenden.",
         "verein-home-card-join-heading": "Uns beitreten.",
         "verein-home-card-join-content": "Unser Projekt lebt von Zusammenarbeit. Wenn Du in der Entwicklung und Verwaltung mitmachen möchtest oder einfach mehr von uns mitbekommen möchtest, heißen wir Dich herzlich bei uns willkommen.",
-        "verein-home-card-join-button": "Ich möchte beitreten."
+        "verein-home-card-join-button": "Ich möchte beitreten.",
+        "footer-tab-website-name": "Datenanfragen.de",
+        "footer-request-generator": "Anfrage generieren",
+        "footer-request-generator-url": "/generator",
```

Comment:
Why are the URLs even i18nable if the menu is hardcoded anyway? Shouldn't simply hardcoding it all do the trick?

Reply:
Nope. Two cases where this doesn't work:

* Some (most?) of the URLs are translated. `/donate` might redirect to `/spenden` but that redirect is ugly and unnecessary.
* For the French version (and potentially others in the future), we need to be able to link to the English sites that aren't translated yet.